frontend/src/metabase/query_builder/components/expressions/ExpressionWidget.tsx

```diff
@@ -52,79 +63,70 @@ const ExpressionWidget = (props: ExpressionWidgetProps): JSX.Element => {
 
   return (
     <ExpressionWidgetWrapper>
-      <div className="p2">
-        <div className="h5 text-uppercase text-light text-bold">{t`Expression`}</div>
-        <div ref={helpTextTargetRef}>
-          <ExpressionEditorTextfield
-            helpTextTarget={helpTextTargetRef.current}
-            expression={expression}
-            name={name}
-            query={query}
-            reportTimezone={reportTimezone}
-            onChange={(parsedExpression: Expression) => {
-              setExpression(parsedExpression);
-              setError(null);
-            }}
-            onError={(errorMessage: string) => setError(errorMessage)}
-          />
-          <p className="h5 text-medium">
-            {t`Think of this as being kind of like writing a formula in a spreadsheet program: you can use numbers, fields in this table, mathematical symbols like +, and some functions. So you could type something like Subtotal - Cost.`}
-            &nbsp;
-            <ExternalLink
-              className="link"
-              target="_blank"
-              href={MetabaseSettings.docsUrl(
-                "questions/query-builder/expressions",
-              )}
-            >{t`Learn more`}</ExternalLink>
-          </p>
-        </div>
+      <ExpressionWidgetExpressionFieldWrapper ref={helpTextTargetRef}>
```

Comment:
nit: Want to hear @mazameli 's opinion about this. The way the new popover is opening is to target around this element (see the highlighted green padding)

<img width="520" alt="Screenshot 2023-03-02 at 1 00 25 PM" src="https://user-images.githubusercontent.com/1937582/222436215-0211709d-4d93-4129-92b7-42f6777ce003.png">

This makes the popover open a bit further from the custom editor input box and make the other text appear above the popover which is a little unpleasant to me
<img width="534" alt="Screenshot 2023-03-02 at 1 00 34 PM" src="https://user-images.githubusercontent.com/1937582/222436506-24791e2b-f76d-4851-8250-08ac1dd22203.png">


Compare to the design which has the popover open closer to the custom editor input box.
<img width="478" alt="image" src="https://user-images.githubusercontent.com/1937582/222436098-1fd5dc4d-48ab-4f9a-8906-1b83a65e9abb.png">

In my opinion, the popover should open closer to the input box similar to the design.

What do you guys think?